modules/ProvBase/Resources/views/Configfile/edit.blade.php

```diff
@@ -106,6 +151,20 @@
             //delete the list
             this.lists.splice(key, 1);
         },
+        ddFilter: function(key) {
+            var search=document.getElementById("ddsearch").value;
+            if (search != "") {
```

Comment:
Just a note: We like to use
```suggestion
            if (search == "") {
                return null;
            }
```
here as it reduces tabs (intendation) in the following lines